_articles/es/how-to-contribute.md

```diff
@@ -335,7 +335,7 @@ Ahora haz lo mismo para los pull requests del proyecto.
 <div class="clearfix mb-4">
   <input type="checkbox" id="cbox13" class="d-block float-left mt-1 mr-2" value="checkbox">
   <label for="cbox13" class="overflow-hidden d-block text-normal">
-    ¿Cu&aacute;n recientemente ocurri&oacute; la entrada (merge) de un pull request? (En GitHub, haz click en el v&iacute;nculo "closed" en la p&aacute;gina de pull requests para ver los PRs cerrados.)
+    ¿Cu&aacute;ndo recientemente ocurri&oacute; la entrada (merge) de un pull request? (En GitHub, haz click en el v&iacute;nculo "closed" en la p&aacute;gina de pull requests para ver los PRs cerrados.)
```

Comment:
Why is rapido (fast) better than recientemente (recently)? The English source is saying the latter.

Reply:
I think the phrase itself makes more sense.

Reply:
It's much more natural to say "Cómo de rápido" or "cuánto tiempo hace desde ..." than not "cómo de reciente ...". It's my point of view.
I think the text would be much more natural with this modification.
I do not think that the meaning of the phrase is changed. After all, what you want to express is how much time has passed since the pull request has been approved.

Reply:
OK, please go ahead and add a commit with the most natural modification!